src/Xamarin.Android.NamingCustomAttributes/Java.Interop/ExportAttribute.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace Java.Interop {
+
+	[Serializable]
+	[AttributeUsage (AttributeTargets.Method | AttributeTargets.Constructor, 
+			AllowMultiple=false, 
+			Inherited=false)]
+#if !JCW_ONLY_TYPE_NAMES
+	public
+#endif  // !JCW_ONLY_TYPE_NAMES
+	partial class ExportAttribute : Attribute {
```

Comment:
Inclusion of this type seems "odd" given that `ExportAttribute` is *also* defined [`Java.Interop/src/Java.Interop.NamingCustomAttributes`](https://github.com/xamarin/java.interop/tree/b58416a5/src/Java.Interop.NamingCustomAttributes/Java.Interop).

Should this copy be removed?